frontend/public/components/instantiate-template.tsx

```diff
@@ -29,12 +29,15 @@ import {
 } from './utils';
 import { SecretModel, TemplateInstanceModel } from '../models';
 import {
-  k8sCreate,
   K8sResourceKind,
   TemplateKind,
   TemplateInstanceKind,
   TemplateParameter,
 } from '../module/k8s';
+import {
+  k8sCreateResource,
+  k8sUpdateResource,
+} from '../../packages/console-dynamic-plugin-sdk/src/utils/k8s';
```

Comment:
We should import other packages via global imports

```suggestion
} from '@console/dynamic-plugin-sdk/src/utils/k8s';
```